pkg/operator/ceph/cluster/cluster.go

```diff
@@ -577,6 +584,59 @@ func (c *cluster) reportTelemetry() {
 	telemetry.ReportKeyValue(c.context, c.ClusterInfo, telemetry.ExternalModeEnabledKey, strconv.FormatBool(c.Spec.External.Enable))
 }
 
+func reportNodeTelemetry(c *cluster) {
+	logger.Info("reporting node telemetry")
+
+	// Report the K8sNodeCount
+	nodelist, err := c.context.Clientset.CoreV1().Nodes().List(c.ClusterInfo.Context, metav1.ListOptions{})
+	if err != nil {
+		logger.Warningf("failed to report the K8s node count. %v", err)
+	} else {
+		telemetry.ReportKeyValue(c.context, c.ClusterInfo, telemetry.K8sNodeCount, strconv.Itoa(len(nodelist.Items)))
+	}
+
+	// Report the cephNodeCount
+	// abc := nodedaemon.CrashCollectorAppName
+	listoption := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", k8sutil.AppAttr, "rook-ceph-crashcollector")}
+	cephNodeList, err := c.context.Clientset.CoreV1().Nodes().List(c.ClusterInfo.Context, listoption)
```

Comment:
What about [this comment](https://github.com/rook/rook/pull/12850#discussion_r1317653156)? I thought we couldn't query the nodes like this to get the daemon count.

Reply:
Addressed